pyntc/devices/iosxewlc_device.py

```diff
@@ -0,0 +1,90 @@
+"""Module for using a Cisco IOSXE WLC device over SSH."""
+import time
+
+from .ios_device import IOSDevice
+from pyntc.errors import RebootTimeoutError, OSInstallError, InstallModeRequired
+
+INSTALL_MODE_FILE_NAME = "packages.conf"
+
+
+class IOSXEWLCDevice(IOSDevice):
+    """Cisco IOSXE WLC Device Implementation."""
+
+    def _wait_for_device_start_reboot(self):
```

Comment:
This one doesn't have a timeout. My thought being we are going to wait until the device reboots no matter how long it could take.  However, I definitely could use the `_wait_for_device_reboot` function.

Reply:
This has been updated. It's not actually different from `_wait_for_device_reboot`.